artemis-core-client/src/main/java/org/apache/activemq/artemis/core/remoting/impl/netty/TransportConstants.java

```diff
@@ -46,6 +46,8 @@ public class TransportConstants {
 
    public static final String HTTP_CLIENT_IDLE_SCAN_PERIOD = "httpClientIdleScanPeriod";
 
+   public static final String NETTY_HTTP_HEADER_PREFIX = "netty.http.header.";
```

Comment:
This still needs to be renamed `nettyHttpHeader.` to be consistent with the other property names.